app/volume.go

```diff
@@ -445,6 +446,10 @@ func (job *Job) backupAndCleanup() (err error) {
 		return err
 	}
 
+	if hasStarted, err := job.waitForBackupProcessStart(BackupProcessStartTimeout); err != nil || !hasStarted {
```

Comment:
? Can it be `err != nil` but `hasStarted`?

Reply:
It cannot. Thank you. Fixed